.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+**Description:**
+<!--- What does this PR do? -->
+
+**Motivation and context:**
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+<!--- If this change depends on another PR, please link to that PR here. -->
+
+**How has this been tested?**
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Reviewers will test your PR in at least this way. -->
+
+**How long should this take to review?**
+<!--- Please estimate if this is a small, average, or large PR. -->
+<!--- Take into account both the size and complexity of the changes. -->
+<!--- Also note if this is a timely PR that should be reviewed by a certain date. -->
+
+**Where should a reviewer start?**
+<!--- Indicate the best part to start looking to understand the changes. -->
+
+**Types of changes:**
+<!--- What types of changes does your code introduce? -->
+<!--- Put an `x` in all the boxes that apply: -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that causes existing functionality to change)
+
+**Checklist:**
+<!--- Go over all the following points. Put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. -->
+<!--- We're here to help! -->
+
+- [ ] I have read the **CONTRIBUTING.rst** document.
+- [ ] My change requires a change to the documentation
+      and I have updated the documentation accordingly.
+- [ ] My change requires a changelog entry and I have included a changelog entry.
```

Comment:
I wonder if we can express these things in a simpler way because it is a bit cumbersome to read "my change requires X and I exactly did X" when the second part doesn't give me any new relevant information as what I have to check. Already the first part tells me that I have to think about whether to do X. Maybe change it to "I did X" and add a note that the creator of the PR should comment on why checkboxes are not checked (e.g. "I believe X is not necessary for this PR, because ...").